packages/server/src/api/controllers/view/utils.js

```diff
@@ -20,8 +20,13 @@ exports.getView = async viewName => {
       return null
     }
 
-    const viewDoc = await db.get(generateMemoryViewID(viewName))
-    return viewDoc.view
+    try {
+      const viewDoc = await db.get(generateMemoryViewID(viewName))
+      return viewDoc.view
+    } catch (err) {
```

Comment:
In these scenarios I always like to check the `err.status === 404` to be sure that this was the cause, in case there are any other unknown errors around reading from couch at the time